sdk/test-utils/perf/runAllTests.ts

```diff
@@ -12,7 +12,7 @@ function spawn(command: string) {
 function runTest(testClassName: string, options: string = "") {
   console.log("\n");
   spawn(
-    `ts-node ./test/index.spec.ts ${testClassName} --warmup 0 --iterations 1 --duration 1 ${options}`
+    `node ./dist-esm/test/index.spec.js ${testClassName} --warmup 0 --iterations 1 --duration 1 ${options}`
```

Comment:
Also add a step here to build the dist-esm files so that this doesn't fail in their absence?

Reply:
Good point. I've added a build step to the package.json script instead, so that it doesn't try and rebuild for each test that gets run.